src/mesh.cpp

```diff
@@ -407,6 +407,13 @@ scene::IMesh* cloneMesh(scene::IMesh *src_mesh)
 
 scene::IMesh* convertNodeboxNodeToMesh(ContentFeatures *f)
 {
+	std::vector<aabb3f> boxes = f->node_box.fixed;
+	return convertNodeboxesToMesh (boxes, NULL);
+}
+
+scene::IMesh* convertNodeboxesToMesh(std::vector<aabb3f> &boxes, f32 *txc)
```

Comment:
In function ‘irr::scene::IMesh* convertNodeboxesToMesh(const std::vector<irr::core::aabbox3d<float> >&, irr::f32*)’:
/home/realbadangel/work2/123/src/mesh.cpp:427:19: error: conversion from ‘std::vector<irr::core::aabbox3d<float> >::const_iterator {aka __gnu_cxx::__normal_iterator<const irr::core::aabbox3d<float>*, std::vector<irr::core::aabbox3d<float> > >}’ to non-scalar type ‘std::vector<irr::core::aabbox3d<float> >::iterator {aka __gnu_cxx::__normal_iterator<irr::core::aabbox3d<float>*, std::vector<irr::core::aabbox3d<float> > >}’ requested
    i = boxes.begin();